packages/notte-core/src/notte_core/credentials/base.py

```diff
@@ -394,95 +268,59 @@ def credentials_dict_to_field(dic: CredentialsDict) -> list[CredentialField]:
 
     @staticmethod
     def credential_fields_to_dict(creds: list[CredentialField]) -> CredentialsDict:
-        dic: CredentialsDict = {}
+        dic: CredentialsDict = {}  # pyright: ignore[reportAssignmentType]
 
         for cred in creds:
             dic[CredentialField.inverse_registry[cred.__class__]] = cred.value
 
         return dic
 
-    def add_credentials_from_env(self, url: str) -> None:
```

Comment:
@leo-notte I added this function because I think it's really usefull from a a user experience perspective. Can we keep it ?